src/components/Main/Modal/Modal.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Modal = (props) => {
```

Comment:
Please use `function` syntax instead.

```suggestion
function Modal(props) {
```